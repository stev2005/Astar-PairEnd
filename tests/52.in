CACCAATCAAACGGAATAGCACCTGTTTAG
15
CTATTAGAAGACAGC
CCGG
A
C
TACTGACCAACCGC
AGTAA
A
GGATGA
ATACTTCAAGACTGTA
CTGCAGTGAACTAG
AGC
CGTGATTTCGA
TTCCTTACTATGCCGCTTA
C
TGGGCTCCGACTCAG
