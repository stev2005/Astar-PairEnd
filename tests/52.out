7
1
0
0
6
2
0
2
6
6
0
4
10
0
7
